hazelcast-sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -0,0 +1,269 @@
+<#--
+// Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+
+/**
+* Parses CREATE EXTERNAL TABLE statement.
+*/
+SqlCreate SqlCreateExternalTable(Span span, boolean replace) :
+{
+    SqlParserPos startPos = span.pos();
+
+    SqlIdentifier name;
+    boolean ifNotExists = false;
+    SqlNodeList columns = SqlNodeList.EMPTY;
+    SqlIdentifier type;
+    SqlNodeList sqlOptions = SqlNodeList.EMPTY;
+    SqlNode source = null;
+}
+{
+    <EXTERNAL> <TABLE>
+    [
+        <IF> <NOT> <EXISTS> { ifNotExists = true; }
+    ]
+    name = SimpleIdentifier()
```

Comment:
Change it to `CompundIdentifier` and validate the schema, if present, is equal to `public`? https://github.com/hazelcast/hazelcast/pull/17266#discussion_r460783679